docs/book/deploying-zenml/zenml-cloud/cloud-system-architecture.md

```diff
@@ -0,0 +1,79 @@
+---
+description: Different variations of the ZenML Cloud architecture depending on your needs.
+---
+
+# ☁ Cloud Architecture
+
+{% hint style="info" %}
+ZenML Cloud is currently in the beta phase, offering users the opportunity to host a managed ZenML instance and gain early access to the exciting new features mentioned above. Beta users will receive priority access to the enhanced functionalities and dedicated support to ensure a smooth onboarding experience. 
+
+[Let us know on Slack](https://zenml.io/slack) if you would like to see other deployment scenarios with ZenML Cloud.
+{% endhint %}
+
+Machine learning often involves data that is sensitive and thus data security is a fundamental requirement. The ZenML Cloud is flexible enough to meet any security requirements, from easy installations to completely airgapped deployments.
+
+The ZenML Cloud consists of the following moving pieces:
+
+* ZenML Cloud API: This is a centralized MLOps control plane that includes a managed ZenML dashboard and a special ZenML server optimized for production MLOps workloads.
```

Comment:
Why do we mention the ZenML Dashboard separate from the ZenML server? Or did you mean the Cloud Dashboard?

Reply:
No i meant the zenml dashboard. why should we not mention it? Its a seperate service